package.json

```diff
@@ -411,11 +411,13 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
+    "vscode:prepublish": "webpack --mode production",
+    "compile": "tsc -p ./",
```

Comment:
why remote watch mode?

Reply:
it is used in "launch tests"?

Reply:
Yes, that's also the use case of CI.

Reply:
So now the test is not compiled in watch mode. Any impact? Will the reload still work?

Reply:
No. We seldom run the test launch profile locally in watch mode in VS Code, as VS Code does not allow running the test with any VS Code instance open.